cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -253,32 +281,23 @@ EOF
 }
 
 function configure_sql_client() {
-  # Configure mysql client to talk to metastore
+  # Configure MySQL client to talk to metastore
   cat <<EOF >/etc/mysql/conf.d/cloud-sql-proxy.cnf
 [client]
 protocol = tcp
-port = ${metastore_proxy_port}
+port = ${METASTORE_PROXY_PORT}
 EOF
```

Comment:
@mikaylakonst Why do we do this only on 0 master?

Reply:
No reason - I guess we assumed the mysql client would only be used on master 0? 

But yeah we should do this for all masters, and we should explicitly set host = master 0 hostname (by default it is localhost). That way you won't have to pass --host and --port to mysql commands

I think we uninstall mysql client on workers (?) so that's probably why we don't do this on them

Reply:
Ack, will create a follow up PR for this then.